### PR TITLE
Player API Updates

### PR DESCRIPTION
**Pull request by katiesweet**
Since it has been a few days since I requested a code review, and no one has reviewed it, I'm requesting to have these changes merged anyway. That way the groups that are managed by @Shemarama @trevorjewkes @michaelkamerath and @adamtew can use this code.

### CODE REVIEW

**Comment by droidkfx** [comment]
I approve this commit.

File: source/PlayerAPI/Card.hpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  CLUBS,
+  DIAMONDS,
+  UNDEFINED
+};
+
+class Card
+{
+private:
+  Suit suit;
+  unsigned int value;
```
Comment (mikamerath) [inline source/PlayerAPI/Card.hpp]: I **strongly** encourage changing this to an enum. It is safe than an int, it communicates intent more clearly than an int, and there are more operations available for enums than for ints. To make sure everyone's card logic doesn't fail, you could do something like TWO =2, THREE = 3, and so on.

**Comment by mikamerath** [comment]
Approved with a strong recommendation to change the ints to an enum. Also, please add unit testing functionality ASAP

**Comment by katiesweet** [comment]
I disagree with you that changing the ints to an enum is necessary. However, I've done it because it is backwards compatable and it was an easy fix. Yes, Ligia and I will work on unit tests as soon as we can.

**Comment by mikamerath** [comment]
That's why it was a strong recommendation. Thanks for doing that 👍

**Pull request merged**